mcs/class/Makefile

```diff
@@ -300,6 +298,11 @@ xbuild_4_0_dirs := \
 	$(xbuild_2_0_dirs)	\
 	Microsoft.Build
 
+ifdef HAVE_BTLS
```

Comment:
We usually do this inside the assembly Makefile. Check e.g. https://github.com/mono/mono/blob/416b6006573d6d517a69faa6a0cc8c1ac1f21607/mcs/class/System.Windows.Forms.DataVisualization/Makefile